.github/workflows/docker.yml

```
@@ -0,0 +1,94 @@
+name: Docker
+
+on:
+  push:
+    branches: [ master ]
+  
+jobs:
+  build:
+    name: Build and test Waltz - postgresql
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:10.6
+        env:
+          POSTGRES_USER: waltz
+          POSTGRES_DB: waltz
+          POSTGRES_PASSWORD: waltz
+        ports:
+          - 5433:5432
+        options: >-
+          --health-cmd "pg_isready -U waltz -d waltz"
+          --health-interval 10s 
+          --health-timeout 5s 
+          --health-retries 10
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
```

As per the [`actions/setup-java` docs](https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#adopt), it is highly recommended to use `temurin` instead.

```suggestion
        distribution: 'temurin'
```